cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -28,6 +28,7 @@ interface IJsonFeatureNames {
   'control-center'?: string;
   dashboard?: string;
   'data-prep'?: string;
+  'enhanced-data-prep'?: string;
```

Comment:
@Biao can you create a flag specific to the feature? May be wrangler-datamodel-viewer? This seems too generic to identify what we are working on.

Reply:
We will have more features (i.e. datamodel search) soon. I was thinking that we can group all the features under one single flag so I used a general name. would wrangler-datamodels work given the context? We can have more discussion during our 1:1 tomorrow. I was planning to discuss it with you tmr but didn't realize this draft pull request also visible to your team. :p

Reply:
@Biao Since this is OSS anyone who can eventually use this feature by turning on the flag. For that the feature should be represented in a straight-forward way. We should definitely have this behind a flag that is specific to datamodel viewer. When we get mappper or search then we can add appropriate flags to those features until they are generally available. Hope that makes sense.

Reply:
I have uploaded another commit. The commit also include files which implement some simple place holder UI. I forgot to include them in my initial commit.